v2g_globals.py

```diff
@@ -20,6 +20,26 @@ def initialize(self):
         c.CHARGER_PLUS_CAR_ROUNDTRIP_EFFICIENCY = efficiency/100
         self.log(f"v2g_globals roundtrip-efficiency: {c.CHARGER_PLUS_CAR_ROUNDTRIP_EFFICIENCY}.")
 
+        power = int(float(self.args["charger_max_charging_power"]))
+        # Make sure this value is between 1440 en 22000
+        tmp = max(min(22000, power), 1440)
+        if power != tmp:
+            self.log(f"Max charge power is changed from {power} to {tmp} to stay within boundries.")
+            power = tmp
+        # set constant so that it can be used in calculations
+        c.CHARGER_MAX_CHARGE_POWER = power
+        self.log(f"v2g_globals max charge power: {c.CHARGER_MAX_CHARGE_POWER}.")
+
+        power = int(float(self.args["charger_max_discharging_power"]))
+        # Make sure this value is between 1440 en 22000
+        tmp = max(min(22000, power), 1440)
+        if power != tmp:
+            self.log(f"Max dis-charge power is changed from {power} to {tmp} to stay within boundries.")
+            power = tmp
+        # set constant so that it can be used in calculations
+        c.CHARGER_MAX_DIS_CHARGE_POWER = power
+        self.log(f"v2g_globals max dis-charge power: {c.CHARGER_MAX_DIS_CHARGE_POWER}.")
```

Comment:
This seems like a very reasonable policy. To help keep the code quality, I suggest refactoring this to a separate function. The initialisation function is getting too lengthy, and these lines serve a single purpose.

Reply:
Will do..